ovn_cluster.sh

```diff
@@ -315,25 +317,29 @@ function start-db-cluster() {
                   --ovn-northd-ssl-ca-cert=${SSL_CERTS_PATH}/pki/switchca/cacert.pem"
     fi
 
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-1 ${OVNCTL_PATH} --db-nb-addr=170.168.0.2 \
-    --db-sb-addr=170.168.0.2 --db-nb-cluster-local-addr=170.168.0.2 \
+    central_1_ip=`cat _ovn_central_1`
```

Comment:
nit... back quotes are so last year! ;)
why not:
```
central_1_ip=$(cat _ovn_central_1)
...
```

Reply:
Done